package.json

```diff
@@ -1,8 +1,13 @@
 {
+  "name": "drink-me",
+  "version": "1.0.2",
   "scripts": {
     "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
+    "push": "yarn version --patch && git push --follow-tags",
```

Comment:
?

Reply:
this increments my project version on push

Reply:
I know, but do you want to increment the version on any push? 

IMO you should increment when you change something, sometime you will push it to repo but your feature will not be completed 

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward-compatible manner
PATCH version when you make backward-compatible bug fixes